packages/webpack/src/loader.ts

```diff
@@ -103,14 +104,16 @@ export default async function vocabLoader(this: LoaderContext) {
 
   const loadedLanguages = Object.keys(loadedTranslation.languages);
 
+  const exportName = findExportNames(source)[0];
   const result = /* ts */ `
     import { createLanguage, createTranslationFile } from '@vocab/webpack/${target}';
 
-    export default createTranslationFile({
+    const translation = createTranslationFile({
       ${loadedLanguages
         .map((lang) => `${JSON.stringify(lang)}: ${renderLanguageLoader(lang)}`)
         .join(',\n')}
     });
+    export { translation as ${exportName} };
```

Comment:
This only affects the code resolved by the loader, the actual generated file hasn't changed.

Reply:
Correct:

https://github.com/seek-oss/vocab/blob/ce755f6140945460299593e27ec7b591ea1f0c22/packages/core/src/compile.ts#L193

Reply:
Wasn't the point to affect the compiled file though, for ✨**rollup reasons**✨?

Reply:
This is looking at the export in the compiled file.

Vocab CLI generates `export default translations`. When Crackle/Rollup compiles the translation file, it will rename the export in the output file. Then the export in the virtual module doesn't match any more.